### PR TITLE
adding top and middle nav

### LINKED ISSUE #37: subtask-1: Header top and middle nav

### PR DESCRIPTION
**Pull request by hparwani2**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #37 

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.page/
- After: https://header-top-middle--sunstar--hlxsites.hlx.page/

- [ ]  If you are adding a new block or a variation to an existing block, has it been added in the [block-inventory](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/Shared%20Documents/sites/sunstar/sunstar-engineering/_drafts/block-inventory.docx?d=w0e883b961c9f4401bae37da23ace83ec&csf=1&web=1&e=KcfAim) ?

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -491,6 +491,83 @@ header .lang-picker > a::after {
...
+header .website-picker:hover > ul {
+  display: block;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+header .website-picker-item > a {
+  font-size: var(--body-font-size-s);
+  font-weight: var(--font-weight-medium);
+  color: var(--white);
+  padding: 11px 0 11px 32px;
```
Comment (sdmcraft) [inline blocks/header/header.css:541]: Nit: Please prefer `rem` units.

File: blocks/header/header.css [hunk 2]
```diff
@@ -491,6 +491,83 @@ header .lang-picker > a::after {
   vertical-align: bottom;
 }
 
+/* stylelint-disable-next-line no-descending-specificity */
+header .website-picker > a::after {
```
Comment (sdmcraft) [inline blocks/header/header.css:495]: The styles look very similar to lang picker. Can we just reuse the same styles? The name can be generalised.

File: blocks/header/header.js [hunk 3]
```diff
@@ -9,22 +9,22 @@ import
...
-    }
+async function decorateWebsitePicker(websitePicker) {
+  websitePicker.classList.add('website-picker');
+  websitePicker.innerHTML = websitePicker.innerHTML.replace(/\[websites\]/, '');
+  const title = 'Sunstar Websites';
+  websitePicker.querySelectorAll(':scope>ul>li').forEach((li) => {
+    li.classList.add('website-picker-item');
   });
+
+  const a = document.createElement('a');
```
Comment (jindaliiita) [inline blocks/header/header.js:20]: I am not sure if its needed or not because the dropdown is not an anchor on the sunstar.com

Comment (hparwani2) [reply blocks/header/header.js:22]: Some elements in top-nav support redirection to different pages like Careers and global network and styles like font size are common for both elements i.e. clickable or unclickable, for a quick fix removed href from <a>

File: blocks/header/header.js [hunk 4]
```diff
@@ -280,7 +279,7 @@ const navDecorators = { 'nav-top': decorateTopNav, 'nav-middle': decorateMiddleN
 export default async function decorate(block) {
   // fetch nav content
   const navMeta = getMetadata('nav');
-  const navPath = navMeta || (getLanguage() === 'en' ? '/nav' : `/${getLanguage()}/nav`);
+  const navPath = navMeta || (getLanguage() === 'en' ? '/_drafts/himanshu/nav' : `/${getLanguage()}/nav`);
```
Comment (jindaliiita) [inline blocks/header/header.js:282]: Change this before merging

**Review by jindaliiita — APPROVED** [review]
LGTM

**Pull request merged**